msg/es_ES.utf8/cubrid.msg

```diff
@@ -185,7 +185,7 @@ $set 5 MSGCAT_SET_ERROR
 116 Base de datos "%1$s" es desconocida, o el archivo "databases.txt" no se puede accesar.
 117 El nombre de ruta absoluto para la base de datos es demasiado largo. La longitud combinada de la ruta "%1$s" y el nombre "%2$s" es %3$d; la longitud combinada debe ser inferior a %4$d.
 118 No se ha podido localizar el directorio de trabajo actual.
-119 Ne eblas trovi ("%1$s") en "%2$s". Bonvolu kontroli permesojn de la dosiero aŭ se estas la gastnomo.
+119 No se puede encontrar el nombre de host ("%1$s") en "%2$s". Verifique los permisos del archivo o si existe el nombre de host.
```

Comment:
For consistency,
its better to use '**hostname**' rather **than** 'host'.

Same is recommended for all msg files.